### PR TITLE
Makes spiders no longer strong against fire/crush. But Are now really strong against stam/clone/breath/tox

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request
Most spiders other then nurses and vipers are now weak to crush/fire but are strong against Stam, clone, breath and tox
## Why It's Good For The Game

Makes fighting spiders more fair on spiders part to not get dunked on by a disabler or toxin based weapons, well still having a reasonable counter of basic melee and combat

## Changelog
:cl:
balance: Spiders have relised that they are meant to be set on fire and crushed to death. Nurses and vipes missed the memo tho. Spiders have also relsied that they are infact toxic shits and no longer get as much damage from toxic, stam and clone based damage.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/giant_spider.dm [hunk 1]
```diff
@@ -36,11 +36,17 @@
 	response_disarm_simple = "gently push aside"
 	response_harm_continuous = "kicks"
 	response_harm_simple = "kick"
+
+	//Health
 	maxHealth = 200
 	health = 200
+	damage_coeff = list(BRUTE = 1.2 BURN = 1.5, TOX = 0.1, CLONE = 0.1, STAMINA = 0.5, OXY = 0.3) //Spiders are infact weak to crush
```
Comment (Trilbyspaceclone) [inline code/modules/mob/living/simple_animal/hostile/giant_spider.dm]: Unironicly spiders are really o2 effective somehow...
_https://www.sciencedaily.com/releases/2011/06/110609105527.htm_

**Comment by silicons** [comment]
i kind of disagree with a nerf here, what's the reasoning of raising their brute/burn damage weakness? if you wanted to do that honestly knock health down a bit instead of playing with armor modifiers. 
plus, spiders really aren't that OP unless someone does the sentient spider meme but that's a whole other issue altogether.
might be good to knock health/maxhealth down to 160 and leave the multipliers alone, while reducing incoming tox/clone/oxy/similar.

also, disablers shouldn't be hurting them.
at all.

**Comment by Trilbyspaceclone** [comment]
I think their hp is fine as is, and I fell that burns should harm spiders more as well as brute well other types like tox/stam/02 shouldnt really harm them at all.
I have no plans on nerfing hp as thats not the goal here its to give them clear weaknesses and clear buffs against bullshit tactics like ***decloner*** and toxic based guns.

**Comment by kevinz000** [comment]
doesn't seem very bullshit.
decloner breaks organic things.
spiders are organic.
i fail to see the problem.
toxic, resistance makes sense but no, this is silly.

**Pull request closed without merging**